network/proto/v1/network.proto

```diff
@@ -41,16 +41,16 @@ message Network {
     // Total rewards emitted on the network in native asset
     BigInt cumulative_rewards = 11;
   
-    // Total number of transactions that have occured on this network
+    // Total number of transactions that have occurred on this network
     BigInt cumulative_transactions = 12;
   
     // Total size of the network in native units (e.g. Bytes in Arweave and Ethereum)
-    BigInt cumulative_size = 13;
+    BigInt cumulative_size  = 13;
```

Comment:
extra space here. Should we get a linter to catch these kind of things?

Reply:
I guess we could put this as a step in the workflow: https://github.com/bufbuild/buf-lint-action. We already need to install the buf cli so I guess it wouldn't take up much more time for the build stage that it is already taking